modules/widgets/src/gimbal-widget.tsx

```diff
@@ -44,14 +45,18 @@ export class GimbalWidget extends Widget<GimbalWidgetProps> {
   }
 
   onRenderHTML(rootElement: HTMLElement): void {
-    const {rotationOrbit, rotationX} = this.getNormalizedRotation();
+    const viewId = this.viewId || Object.values(this.viewports)[0]?.id || 'default-view';
```

Comment:
Nit (and not introduced by this PR), the rendering doesn't maintain a solid line due to using `transform()` - it might look better to modify the radii of the ellipse drawn directly

<img width="68" height="52" alt="Screenshot 2025-09-26 at 11 35 01" src="https://github.com/user-attachments/assets/de1b48c7-ae5e-46ec-a622-aa78fbbc886c" />